portal/tokenList/index.ts

```diff
@@ -30,31 +30,60 @@ const extendWithWhiteList = <
     }
   }
 
+const normalizeAddress = (address: string) =>
+  isAddress(address, { strict: false }) ? getAddress(address) : address
```

Comment:
now that I see that this function is also used in here https://github.com/hemilabs/ui-monorepo/pull/1625/files#diff-383d7e26309039ac1b2ead79a8fad509987e8d43fb43da4d0f5b8314025cc088R22

perhaps we can create a `utils/address.ts` with this function. What do you think?